src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -271,11 +274,19 @@
 helps['feature list'] = """
 type: command
 short-summary: List preview features.
+examples:
+-   name: List preview features.
+    text: az feature list --namespace {namespace}
+    crafted: true
```

Comment:
Unless all features are preview features, this description is incorrect. Also, it would be better to include an example of a namespace rather than just a placeholder.